bpf/lib/common.h

```diff
@@ -445,6 +446,16 @@ enum {
  */
 #define MARK_MAGIC_SNAT_DONE		0x1500
 
+/* MARK_MAGIC_IPIP_DONE can overlap with MARK_MAGIC_SNAT_DONE with both being
+ * mutual exclusive.
```

Comment:
This is used to ...

Reply:
So the IPIP dispatch requires DSR and for this we do not enable NAT code hence mutually exclusive.

Reply:
Sorry comment was short, I was just hinting that maybe the comment could have a bit extra detail about how the define is used.

Reply:
I've updated the commit message with more details, but I could also extend the comment slightly. Can do after CI went through before merging.